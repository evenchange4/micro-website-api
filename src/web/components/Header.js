import * as React from 'react';
import styled from 'styled-components';
import TweenOne from 'rc-tween-one';

const Wrapper = styled.header`
  font-size: 30px;
  text-align: center;
  margin-top: 24px;
  margin-bottom: 24px;
  font-family: 'Open Sans Condensed', sans-serif;
`;

const Header = () => (
  <TweenOne
    animation={{
      opacity: 1,
      duration: 800,
      y: 0,
    }}
    style={{
      opacity: 0,
      transform: 'translateY(-40px)',
    }}
    component={Wrapper}
  >
    Website API
  </TweenOne>
);

export default Header;
