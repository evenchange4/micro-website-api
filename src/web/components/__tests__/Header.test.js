import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from '../Header';

it('should render <Header> correctly', () => {
  const wrapper = mount(<Header />);

  expect(toJson(wrapper)).toMatchSnapshot();
});
