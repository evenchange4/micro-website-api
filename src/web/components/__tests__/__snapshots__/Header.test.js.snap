// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render <Header> correctly 1`] = `
<Header>
  <mock-tween-one
    animation={
      Object {
        "duration": 800,
        "opacity": 1,
        "y": 0,
      }
    }
    component={[Function]}
    style={
      Object {
        "opacity": 0,
        "transform": "translateY(-40px)",
      }
    }
  >
    Website API
  </mock-tween-one>
</Header>
`;
